worker/dbaccessor/worker.go

```diff
@@ -266,6 +266,41 @@ func (w *dbWorker) Wait() error {
 	return w.catacomb.Wait()
 }
 
+// Report provides information for the engine report.
+func (w *dbWorker) Report() map[string]any {
+	w.mu.RLock()
+	defer w.mu.RUnlock()
+
+	// We need to guard against attempting to report when setting up or dying,
```

Comment:
will it be obvious in the engine report this is happening? should different output happen? `leader: ""` is ambiguous

Reply:
There isn't anything I'd put in otherwise. You either have an address or you don't. Anything else might confuse matters.